ai/ollama/v0/config/setup.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": true,
+  "properties": {
+    "endpoint": {
```

Comment:
@namwoam 
When the users deploy their own Ollama server, we will need users to set up their API auth.